src/assets/styles/main.scss

```diff
@@ -0,0 +1,21 @@
+@font-face {
+	font-family: 'Source Sans Pro';
+	font-style: normal;
+	font-weight: 400, 600, 700;
+	font-stretch: 100%;
+	font-display: swap;
+	src: url('/../public/fonts/SourceSansPro-Regular.ttf'), url('/../public/fonts/SourceSansPro-SemiBold.ttf'),
```

Comment:
use absolute paths. 
for public it will be fonts/Sour...

Reply:
It doesnt work